lua/tardis/cl_options.lua

```diff
@@ -96,70 +96,126 @@ function TARDIS:ChangeOption(id,data)
     frame:MakePopup()
 end
 
-TARDIS:AddGUISetting("Options", function(self,frame,screen)
-    local options={}
-    local sections={}
-    for k,v in pairs(TARDIS:GetSettingsData()) do
-        if v.option then
-            table.insert(options,{v.name,v.id,v})
-            if v.section and not table.HasValue(sections,v.section) then
-                table.insert(sections,v.section)
+function TARDIS:CreateOptionInterface(id, data)
+
+    local text = data.name or id
+    local tooltip = data.desc or ""
+    if data.value ~= nil then
+        tooltip = tooltip .. "\n\nDefault: " .. tostring(data.value)
```

Comment:
![изображение](https://user-images.githubusercontent.com/32988384/160284363-0c564de3-9d23-43cb-9dd7-183ff3b19ed8.png)
Fixed

Reply:
Do we need to show the number here as well? Just showing 'Default: In-built notifications' would be perfect imo

Reply:
It's value + text
Values are not required here, but might be useful in other situations
Plus, it was easier to add (I just added the text to existing tip, instead of making the tip conditional

Reply:
How much do you want this to change?
(tbh, I'm lazy to waste too much time on perfectionism noone would even notice)

Reply:
Not that bothered was mostly just curious, works for me!